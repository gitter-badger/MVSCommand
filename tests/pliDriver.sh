# here is where the PL/I compiler is: https://linuxvm:19443/RDT/edit/edit.html#/ZOS/MVSFiles/10.1.1.2:2016/IEL450.SIBMZPRC/IBMZC
tso alloc dsn\("TEST.PLI.SYSUT1"\) dsorg\(ps\) catalog tracks space\(100,10\)
(export STEPLIB=IEL450.SIBMZCMP:CEE.SCEERUN; mvscmd --pgm=ibmzpli --sysprint=* --sysout=* --sysin=ibmuser.test.pli\(main\) --syslin=ibmuser.test.obj\(main\) --sysut1=ibmuser.test.pli.sysut1 | awk '!/15655-W67|0 Compiler/' )
tsocmd delete "TEST.PLI.SYSUT1"
