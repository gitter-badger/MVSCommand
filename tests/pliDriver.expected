alloc dsn(TEST.PLI.SYSUT1) dsorg(ps) catalog tracks space(100,10)
-                     Options Specified
0  Install:
0  Command:
0  Install:
- Compiler Messages
0 Message       Line.File Message Description
0 IBM1085I W      35.0    TOK may be uninitialized when used.
- File Reference Table
0   File    Included From  Name
0      0                   IBMUSER.TEST.PLI(MAIN)
- Component    Return Code    Messages (Total/Suppressed)    Time
0 End of compilation of PLICALC
delete TEST.PLI.SYSUT1
IDC0550I ENTRY (A) IBMUSER.TEST.PLI.SYSUT1 DELETED
